lib/models/ObjectMD.js

```diff
@@ -934,6 +935,26 @@ class ObjectMD {
     getValue() {
         return this._data;
     }
+
+    /**
+     * Returns the isAborted flag
+     *
+     * @returns {string} isAborted
+     */
+    isAborted() {
```

Comment:
```suggestion
    getIsAborted() {
```

For consistency, e.g. with `getIsDeleteMarker`, but also because the unit tests expect getters to start with `get` (https://github.com/scality/Arsenal/pull/1665/files#diff-5c480243ad2c4eed29b2608de22be2f29e80ebe70772cb7ad7b4ab22d199940aR115)

Which also means there should be a unit test for this getter/setter :slightly_smiling_face: